### PR TITLE
Update ru_ru.json

### LINKED ISSUE #164: Update to Russian translate
Russian traslate is very old and i make own translate update for this language. 
[ru_ru.zip](https://github.com/Noaaan/MythicMetals/files/11454600/ru_ru.zip)

### PR DESCRIPTION
**Pull request by Noaaan**
Co-authored-by: HalloSenpai

Closes #164

### CODE REVIEW

File: src/main/resources/assets/mythicmetals/lang/ru_ru.json [hunk 1]
```diff
@@ -1,682 +1,857 @@
...
+  "block.alloy_forgery.adamantite_block_forge_controller": "Адаманитовый контроллер плавильни",
+  "block.mythicmetals.adamantite_anvil": "Адаманитовая наковальня",
+  "block.mythicmetals.adamantite_block": "Адаманитовый блок",
+  "block.mythicmetals.adamantite_ore": "Адаманитовая руда",
+  "block.mythicmetals.aquarium_anvil": "Аквариевая наковальня",
+  "block.mythicmetals.aquarium_block": "Аквариевый блок",
+  "block.mythicmetals.aquarium_ore": "Аквариевая руда",
   "block.mythicmetals.banglum_anvil": "Банглиевая наковальня",
+  "block.mythicmetals.banglum_block": "Банглиевый блок",
+  "block.mythicmetals.banglum_nuke_core": "Ядро Банглиевой бомбы",
```
Comment (BasiqueEvangelist) [inline src/main/resources/assets/mythicmetals/lang/ru_ru.json:11]: why is "Банглиевой" capitalized here

File: src/main/resources/assets/mythicmetals/lang/ru_ru.json [hunk 2]
```diff
@@ -1,682 +1,857 @@
...
+  "block.mythicmetals.banglum_ore": "Банглиевая руда",
+  "block.mythicmetals.banglum_tnt": "Банглиевый TNT",
+  "block.mythicmetals.blackstone_stormyx_ore": "Чернитовая Штормиксовая руда",
   "block.mythicmetals.bronze_anvil": "Бронзовая наковальня",
+  "block.mythicmetals.bronze_block": "Бронзовый блок",
+  "block.mythicmetals.calcite_kyber_ore": "Кальцинированная Кьюбер руда",
```
Comment (BasiqueEvangelist) [inline src/main/resources/assets/mythicmetals/lang/ru_ru.json:17]: Imo "Кайбер" is better here, as it actually reflects the pronounciation in English

File: src/main/resources/assets/mythicmetals/lang/ru_ru.json [hunk 3]
```diff
@@ -1,682 +1,857 @@
...
+
+  "abilities.mythicmetals.aqua_affinity": "Естественное Аква Аффинити",
+  "abilities.mythicmetals.blast_mining": "Взрывной майнинг",
+  "abilities.mythicmetals.blast_padding": "Взрывная прокладка",
+  "abilities.mythicmetals.blast_protection": "Естественная Взрывоустойчивость",
+  "abilities.mythicmetals.bonus_fortune": "Бонус Удачи",
+  "abilities.mythicmetals.bonus_looting": "Бонус Добычи",
+  "abilities.mythicmetals.carmot_shield": "+5 Кармотового щита",
+  "abilities.mythicmetals.depth_strider": "Естественный Глубинный страйдер",
+  "abilities.mythicmetals.hot": "Брендинг",
```
Comment (BasiqueEvangelist) [inline src/main/resources/assets/mythicmetals/lang/ru_ru.json:612]: Branding doesn't mean this in this context. "Клеймение" would probably fit better here.

File: src/main/resources/assets/mythicmetals/lang/ru_ru.json [hunk 4]
```diff
@@ -1,682 +1,857 @@
...
+  "item.mythicmetals.durasteel_helmet": "Дюрасталевый шлем",
+  "item.mythicmetals.durasteel_hoe": "Дюрасталевая мотыга",
+  "item.mythicmetals.durasteel_ingot": "Дюрасталевый слиток",
+  "item.mythicmetals.durasteel_leggings": "Дюрасталевые поножи",
+  "item.mythicmetals.durasteel_engine": "Дюрасталевый двигатель",
+  "item.mythicmetals.durasteel_nugget": "Дюрасталевый кусочек",
+  "item.mythicmetals.durasteel_pickaxe": "Дюрасталевая кирка",
+  "item.mythicmetals.durasteel_shovel": "Дюрасталевая лопата",
+  "item.mythicmetals.durasteel_sword": "Дурасталевый меч",
+  "item.mythicmetals.froge": "Froge",
```
Comment (BasiqueEvangelist) [inline src/main/resources/assets/mythicmetals/lang/ru_ru.json:221]: Сколько еще будет безжабных сред?

**Review by BasiqueEvangelist — COMMENTED** [review]
Didn't fully review it yet, here are some comments

File: src/main/resources/assets/mythicmetals/lang/ru_ru.json [hunk 5]
```diff
@@ -1,682 +1,857 @@
...
+  "advancement.mythicmetals.star_platinum.description": "Изготовьте Звёздную Платину из Старрита и Платины",
+  "advancement.mythicmetals.star_platinum.title": "Самый Биззарный металл в Мире",
+  "advancement.mythicmetals.starrite.description": "Найдите Старрит. Необычный кристал, который можно найти в Энде",
+  "advancement.mythicmetals.starrite.title": "Павшие звездопады",
+  "advancement.mythicmetals.steel.description": "Изготовьте Сталь. Простой сплав из Железа и Марганца",
+  "advancement.mythicmetals.steel.title": "Но сталь тяжелее чем...",
+  "advancement.mythicmetals.stormyx.description": "Найдите Штормикс в глубинах Незера",
+  "advancement.mythicmetals.stormyx.title": "Магические металлы",
+  "advancement.mythicmetals.stormyx_shell.description": "Получите Штормиксовую оболчку, редко выпадающую из Штормиксовой руды ",
+  "advancement.mythicmetals.stormyx_shell.title": "Назаровские Алмазы",
```
Comment (BasiqueEvangelist) [inline src/main/resources/assets/mythicmetals/lang/ru_ru.json:594]: Pretty sure Vanilla calls the Nether "Незер"

**Comment by BasiqueEvangelist** [comment]
@HalloSenpai forgot to ping you :trollface:

**Comment by HalloSenpai** [comment]
> @HalloSenpai forgot to ping you :trollface:
i can fix it next updates

**Comment by Noaaan** [comment]
Bumping this PR. Would be nice if you could re-target this towards the 1.20 branch, as generally 1.19.4 support is at the end of its support-cycle.

**Comment by HalloSenpai** [comment]
> Bumping this PR. Would be nice if you could re-target this towards the 1.20 branch, as generally 1.19.4 support is at the end of its support-cycle.

Im very busy IRL. Sorry for this. I have first paid work. That's amazing for me

**Comment by Noaaan** [comment]
No worries, congratulations on your first job!

**Comment by Noaaan** [comment]
Bumping this translation, as it has not been updated in a while

**Comment by Noaaan** [comment]
Considering this is outdated I will go ahead and close this PR. Since then I have managed to get my hands on some russian localization. There might be some missing strings or issues with it, so feel free to correct anything in the current lang that might be wrong.

**Pull request closed without merging**